vowpalwabbit/core/src/vw.cc

```diff
@@ -315,8 +323,11 @@ std::unique_ptr<VW::workspace> VW::seed_vw_model(VW::workspace& vw_model, const
 
   auto options = VW::make_unique<config::options_cli>(serialized_options);
 
-  auto new_model = VW::initialize(std::move(options), nullptr, true /* skip_model_load */, driver_output_func,
-      driver_output_func_context, custom_logger);
+  std::unique_ptr<config::options_i, options_deleter_type> options_custom_deleter(
+      new VW::config::options_cli(serialized_options), [](VW::config::options_i* ptr) { delete ptr; });
+
+  auto new_model = initialize_internal(std::move(options_custom_deleter), nullptr, false /* skip model load */,
```

Comment:
Is skip_model_load=true intended?

Reply:
Yes, we don't want to skip loading the model the user supplied